Dry-run identified changes to `rstudio-login.md` (#202)
AlexsLemonade/training-modules: virtual-setup/rstudio-login.md

<br>

Press `Enter`. Then type in the password you were given in Slack.
Keep in mind, as you are typing in your password, it will remain blank, but it is receiving what you are typing.

<img src = "screenshots/rstudio-password-change-2.png" width = 400>
<br>
<br>

Type in the new password you've chosen once, and press `Enter`, then it will ask you to confirm by typing it in again.
Again, no text or dots will show as you are typing your new password.
Also press `Enter`.

> If you forget your password at anytime, Slack on of the CCDL team members to assist you.

Reviewer: ```suggestion
If you forget your password at any time, Slack on of the CCDL team members to assist you.
```

Reviewer 2: ```suggestion
If you forget your password at any time, Slack one of the CCDL team members to assist you.
```

Reviewer: jinx!

Reviewer 2: :sonic:

<br>
<br>

> RStudio may ask you if you would like to save your current workspace?

Author: ```suggestion
RStudio may ask you if you would like to save your current workspace.
```

Reviewer: Reviewing yourself again?

> RStudio may ask you if you would like to save your current workspace.

Reviewer: We may need to add one more screenshot here, for when there are multiple files with changes. If they haven't saved their notebook yet, they may get a different dialog box, where we would want them to save the notebook, but not `.Rdata`

> In general, we advise saying `Don't Save` so you can start fresh in your next session (this will save your objects from the R environment to a file and reload them, more on the R environment later).

Reviewer: I think the parenthetical is more confusing than enlightening here. What we are advising them to do is *not* save. Alternatively, change "this will" to "Choosing 'Save' at this point would"
```suggestion
In general, we advise choosing `Don't Save` so you can start fresh in your next session.
```

Author: I kept rewriting these sentences. Because I don’t want to be confusing but also want them to know what it means.

<br>
<br>

If you've made changes across multiple files, RStudio will ask you if you'd like to save your other files' changes.

> Check off which files' changes you'd like to keep.

Reviewer: ```suggestion
Check off which files' changes you'd like to keep.
Note again that we recommend *not* saving the `.Rdata` Workspace file.
```

> <img src = "screenshots/rstudio-multiple-changes.png" width = 300>

Reviewer: Can you make a version of this screenshot *without* `.Rdata` checked?

Author: Ah good point.

We go into more detail on the R environment and other RStudio navigating tidbits in our [guide to RStudio](../intro-to-R-tidyverse/00a-rstudio_guide.md) as well as our [first intro to R notebook](../intro-to-R-tidyverse/01-intro_to_base_R.Rmd).

As always, please reach out to our CCDL team through Slack if you have any questions!

## A troubleshooting note:

A not uncommon problem, upon trying to re-login to RStudio Server, is to see this screen:

<img src = "screenshots/unauthorized.png" width = 300>
<br>
<br>

> What will *usually* remedy this, is quitting your internet browsing application completely and restarting it.

Reviewer: ```suggestion
Quitting your internet browser completely and restarting it will *usually* remedy this.
```
